change/@ni-nimble-blazor-a1bc134a-f028-4243-b969-8fb7faf901d6.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Upgrade to Angular 13.",
```

It feels incongruous for the NimbleBlazor change log to say this change is upgrading Angular. How about a more targeted comment for the changes that impact this package?
```suggestion
  "comment": "Update `@ni/eslint-config-javascript` dependency version.",
```